src/multi.c

```diff
@@ -326,8 +335,12 @@ void watchCommand(client *c) {
         addReplyError(c,"WATCH inside MULTI is not allowed");
         return;
     }
+    if (c->flags & CLIENT_DIRTY_CAS)
```

Comment:
@soloestoy I think it is just style issue.
but I think if we change this condition like below

```
if (c->flags & CLIENT_DIREY_CAS != CLIENT_DIREY_CAS) {
      ......
}
```

we can remove watch_done label. if we use this label only once and here. I think just changing condition is better to understand. :) 
What do you think?

Reply:
Yes... using `goto` here is not a good code style, thanks~ updated.